src/main/resources/static/javascript/mainApp/general.controller.js

```diff
@@ -265,6 +267,23 @@
         $scope.hoverCopy = () => {
             $("[data-content='copy']").popover();
         };
+
+        /**
+         * Updates the active user profile, clears cache by reloading the page with a cache-busting URL.
+         * @param profile - the type of profile to switch to ('USER', 'OWNER', 'ADMIN')
+         */
+        $scope.updateActiveUserProfile = function(profile) {
+            groupingsService.updateOotbActiveProfile(profile, () => {
+                $scope.$apply(function() {
```

Comment:
Make sure to use arrow functions as it's the style of the rest of the code:
```
$scope.updateActiveUserProfile = (profile) => {
            groupingsService.updateOotbActiveProfile(profile, () => {
                $scope.$apply(() => {
```

Reply:
Thanks!